op-node/rollup/derive/l1_block_info.go

```diff
@@ -36,6 +38,11 @@ const (
 	RegolithSystemTxGas = 1_000_000
 )
 
```

Comment:
Can you add a comment here saying that the hardcoded chain ids are for the first devnet release? Can you make 3 of them - 10666, 10777, 10888 ? We will spin up the devnet with 3 chains with these chainids